src/sentry/incidents/endpoints/organization_incident_index.py

```diff
@@ -34,6 +35,14 @@ def get(self, request, organization):
         if query_alert_rule is not None:
             incidents = incidents.filter(alert_rule=query_alert_rule)
 
+        query_start = request.GET.get("start")
+        if query_start is not None:
+            incidents = incidents.filter(start__gte=query_start)
+
+        query_end = request.GET.get("end")
+        if query_end is not None:
+            incidents = incidents.filter(end__gte=query_end)
```

Comment:
We still need tests to make sure this works

Reply:
Added them